db/migrate/20200513184343_create_user_questions_answers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUserQuestionsAnswers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_questions_answers do |t|
+      t.references :question, foreign_key: true, index: true
+      t.references :answer, foreign_key: true, index: true
```

Comment:
So what can be the use-case of having index on `answer` field ?

Reply:
No use. Removed it. Thanks